### PR TITLE
Support latest java 11 version + java 17

### PR DESCRIPTION
**Pull request by vik4es**
Added new version of java 11 + java 17

@BaCaRoZzo Can you take a look, please?

### CODE REVIEW

File: bin/install [hunk 1]
```diff
@@ -94,6 +94,18 @@ get_java() {
...
+			url=${corretto_base}
+			signature=${corretto_base}.sig
+			sigkey="A122542AB04F24E3"
+    ;;
+    corretto-17.0.1.12.1)
+            local corretto_base="https://corretto.aws/downloads/resources/$(cut -d - -f2 <<< "$version")/amazon-$version-linux-x64.tar.gz"
+			url=${corretto_base}
+			signature=${corretto_base}.sig
+			sigkey="A122542AB04F24E3"
+    ;;
```
Comment (BaCaRoZzo) [inline bin/install]:
```suggestion
        corretto-11.0.13.8.1)
            local corretto_base="https://corretto.aws/downloads/resources/$(cut -d - -f2 <<< "$version")/amazon-$version-linux-x64.tar.gz"
			url=${corretto_base}
			signature=${corretto_base}.sig
			sigkey="A122542AB04F24E3"
        ;;
    	corretto-17.0.1.12.1)
            	local corretto_base="https://corretto.aws/downloads/resources/$(cut -d - -f2 <<< "$version")/amazon-$version-linux-x64.tar.gz"
			url=${corretto_base}
			signature=${corretto_base}.sig
			sigkey="A122542AB04F24E3"
        ;;
```

**Review by BaCaRoZzo — COMMENTED** [review]
Afaict this is correct. Just fix the indentation please.

File: bin/install (hunk 1 above)
Comment (vik4es) [reply bin/install]: @BaCaRoZzo I've fixed the indentation for the previous 2 sections. Now all branches have the same indentation. Wdyt?

Comment (vik4es) [reply bin/install]: Hm. Something wrong with IDE... Fixing

Comment (vik4es) [reply bin/install]: @BaCaRoZzo Fixed.

**Review by BaCaRoZzo — APPROVED** [review]
Looks good to me.
Files are there and signature stuff should be the same for `aws` urls.

**Comment by vik4es** [comment]
The link looks correct. The same form Github releases:
https://corretto.aws/downloads/resources/17.0.1.12.1/amazon-corretto-17.0.1.12.1-linux-x64.tar.gz
Signature key is the same for all versions;

**Comment by BaCaRoZzo** [comment]
@vik4es should I merge it for you?

**Comment by vik4es** [comment]
@BaCaRoZzo Yes, please. I don't have permissions for merge :slightly_smiling_face:

**Pull request merged**